samples/open-telemetry/tracing/README.md

```diff
@@ -0,0 +1,111 @@
+# Open Telemetry Tracing
+
+This sample demonstrates the support for the OpenTelemetry tracing provider with the Telemetry API.
+
+## Start otel-collector service
+
+First, deploy the `otel-collector` backend with simple configuration.
+
+```bash
+kubectl apply -f ../otel.yaml -n istio-system
+```
+
+The otel-collector will create a grpc receiver on port `4317`, and later the sidecars will report trace information to this grpc port. You can find more details from [here](https://github.com/open-telemetry/opentelemetry-collector).
+
+The receiver is defined as the following configuration:
+
+```yaml
+receivers:
+  otlp:
+    protocols:
+      grpc:
+      http:
+processors:
+  batch:
+exporters:
+  logging:
+    loglevel: debug
+service:
+  pipelines:
+    logs:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [logging]
+```
+
+In this example, `zipkin` is the exporter for gathering the traces, which is defined as:
+
+```yaml
+exporters:
+  zipkin:
+    # Export to zipkin for easy querying
+    endpoint: http://zipkin.istio-system.svc:9411/api/v2/spans
+service:
+  pipelines:
+    logs:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [logging]
+    traces:
+      receivers:
+      - otlp
+      - opencensus
+      exporters:
+      - zipkin # add the exporter here
+      - logging
+```
+
+If you have not deployed the `zipkin` service, you can use the following command:
+
+```bash
+kubectl apply -f ../../addons/extras/zipkin.yaml -n istio-system
```

Comment:
Could this be Jaeger too?

Reply:
+1, we have made jaeger the "default" sample for tracing, I would recommend using it here. I don't recall why we did it (maybe flipped a coin) but it seems simpler to be consistent

Reply:
I've tried Jaeger too, and both of them work. I'll switch to Jaeger for consistence